src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_SupportUserBanner.cshtml

```diff
@@ -0,0 +1,30 @@
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.SupportUserBannerViewModel
+<div id="service-console-header" class="header-wrapper">
+    <div class="header-global">
+        <div class="header-logo">
+            <p class="heading-large">
+                Apprenticeship Service Console
+            </p>
+        </div>
+    </div>
+    <div class="header-proposition">
+        <div class="content">
+            Service user <span>(T2 Support)</span>
+        </div>
+    </div>
+
+
+</div>
+
+<div id="service-console-msg">
+
+    <div id="global-header-bar"></div>
+
+    <div class="header-wrapper">
+
+        <div class="content">
+            <p>You are viewing as @(Model.Account?.Name ?? string.Empty) (account ID:  @(Model.Account?.PublicHashedId ?? string.Empty))</p>
```

Comment:
Why are we checking for null here, are we expecting `Account` to not be populated? However we have safe checks in the controller action that will assure that we render this view only when applicable. Feels like I may have missed something here.

Reply:
When EmployerFinance uses this then the account will be null and an error would be thrown.  This partial shouldn't make assumptions about what calls it.